### DISCUSSION TITLE
Missing dropdown for all open tabs

### DISCUSSION THREAD

**Discussion by rtqate** [rafaelmardojai/firefox-gnome-theme#732] [Ideas]
This might be a bug as well, but I couldn't find it there... Normally, Firefox has a dropdown, shaped as a down-arrow, which lists all open tabs. (It's on the right side, near the "close" button.) 
I do have `browser.tabs.tabmanager.enabled` under `about:config` set to `true`, but, somehow, this doesn't seem to help. 

![all_tabs_downarrow](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/90438548/85561e75-5e1a-4d6a-bf0f-538227eed57f)

**Comment by rafaelmardojai** [comment]
Yes, as a design decision the all tabs menu only appears when the tabbar is overflowing.

**Reply by benyaminl** [reply]
Is there anyway to disable it fully using CSS? 

Thanks you
